datastore/src/main/java/com/ds/datastore/Book.java

```diff
@@ -28,6 +28,15 @@
     private Long storeID;
 
     public Book() {}
+    public Book(Book book)
```

Comment:
why is this necessary?

Reply:
Since java is pass by reference for the one to many stores method we wanted to add the same book with different store ids to a list, in order to make copies without getting and setting each variable, we now have a constructor which creates it

Reply:
^From Seth

Reply:
👍